package exceptions

func Error(m string, err error) {
	println(err)
	println(m)
}
